ios/Capacitor/Capacitor/WebViewAssetHandler.swift

```diff
@@ -25,89 +27,97 @@ open class WebViewAssetHandler: NSObject, WKURLSchemeHandler {
+                    if streamParts.count > 1 {
+                        toRange = Int(streamParts[1]) ?? toRange
                     }
-                }
-                let urlResponse = URLResponse(url: localUrl, mimeType: mimeType, expectedContentLength: data.count, textEncodingName: nil)
-                let httpResponse = HTTPURLResponse(url: localUrl, statusCode: 200, httpVersion: nil, headerFields: headers)
-                if isMediaExtension(pathExtension: url.pathExtension) {
-                    urlSchemeTask.didReceive(urlResponse)
+                    let rangeLength = toRange - fromRange + 1
+                    try fileHandle.seek(toOffset: UInt64(fromRange))
+                    data = fileHandle.readData(ofLength: rangeLength)
+                    headers["Accept-Ranges"] = "bytes"
+                    headers["Content-Range"] = "bytes \(fromRange)-\(toRange)/\(totalSize)"
+                    headers["Content-Length"] = String(data.count)
+                    let response = HTTPURLResponse(url: localUrl, statusCode: 206, httpVersion: nil, headerFields: headers)
+                    tasks.with(urlSchemeTask) { $0.didReceive(response!) }
+                    try fileHandle.close()
                 } else {
-                    urlSchemeTask.didReceive(httpResponse!)
+                    if !stringToLoad.contains("cordova.js") {
+                        if isMediaExtension(pathExtension: url.pathExtension) {
+                            data = try Data(contentsOf: fileUrl, options: Data.ReadingOptions.mappedIfSafe)
+                        } else {
+                            data = try Data(contentsOf: fileUrl)
+                        }
+                    }
+                    let urlResponse = URLResponse(url: localUrl, mimeType: mimeType, expectedContentLength: data.count, textEncodingName: nil)
+                    let httpResponse = HTTPURLResponse(url: localUrl, statusCode: 200, httpVersion: nil, headerFields: headers)
+                    tasks.with(urlSchemeTask) {
+                        if isMediaExtension(pathExtension: url.pathExtension) {
+                            $0.didReceive(urlResponse)
+                        } else {
+                            $0.didReceive(httpResponse!)
+                        }
+                        $0.didReceive(data)
+                    }
                 }
+
+            } catch let error as NSError {
+                tasks.with(urlSchemeTask) { $0.didFailWithError(error) }
+                return
             }
-            urlSchemeTask.didReceive(data)
-        } catch let error as NSError {
-            urlSchemeTask.didFailWithError(error)
-            return
+            tasks.with(urlSchemeTask) { $0.didFinish() }
         }
-        urlSchemeTask.didFinish()
     }
 
     open func webView(_ webView: WKWebView, stop urlSchemeTask: WKURLSchemeTask) {
-        CAPLog.print("scheme stop")
+        print("cancelling task \(urlSchemeTask.hashString)")
```

Comment:
```suggestion
```
remove print

with latest changes the app sometimes doesn't start, I just get a white screen and no messages after `Loading app at capacitor://localhost...`